buildenv/jenkins/docker-slaves/ppc64le/ubuntu16/Dockerfile

```diff
@@ -40,36 +40,41 @@ RUN apt-get update \
   && apt-get install -qq -y --no-install-recommends \
     software-properties-common \
     python-software-properties \
-  && add-apt-repository ppa:ubuntu-toolchain-r/test \
   && apt-get update \
   && apt-get install -qq -y --no-install-recommends \
     ant \
     ant-contrib \
     autoconf \
-    build-essential \
+    binutils \
     curl \
-    g++-4.8 \
-    gcc-4.8 \
-    gdb \
     git \
+    libc6-dev \
     make \
     openssh-client \
     openssh-server \
     perl \
     ssh \
     wget \
     xvfb \
+    xz-utils \
   && rm -rf /var/lib/apt/lists/*
 
-# Install Docker module to run test framework
-RUN echo yes | cpan install JSON Text::CSV
+# Install GCC-7.3
+RUN cd /usr/local \
+  && wget -O gcc-7.tar.xz "https://ci.adoptopenjdk.net/userContent/gcc/gcc730+ccache.ppc64le.tar.xz" \
```

Comment:
Using these pre-compiled you will need to also add libstdc++.so* to either the LD_LIBRARY_PATH or set it up with ldconfig.  Same applies to other instance below.

Reply:
I added the libstdc++.so* from the precompiled binaries to ldconfig